doc/en/user/source/configuration/virtual-services.rst

```diff
@@ -78,51 +82,86 @@ The part of interest in the above request is the "topp" prefix to the wfs servic
    
       <FeatureType xmlns:topp="http://www.openplans.org/topp">
        <Name>topp:tasmania_cities</Name>
-    --
+       ...
       <FeatureType xmlns:topp="http://www.openplans.org/topp">
        <Name>topp:tasmania_roads</Name>
-    --
+       ...
       <FeatureType xmlns:topp="http://www.openplans.org/topp">
        <Name>topp:tasmania_state_boundaries</Name>
-    --
+       ...
       <FeatureType xmlns:topp="http://www.openplans.org/topp">
        <Name>topp:tasmania_water_bodies</Name>
-    --
+       ...
       <FeatureType xmlns:topp="http://www.openplans.org/topp">
        <Name>topp:states</Name>
-       
+       ...
     </wfs:WFS_Capabilities>
 
-The above feature types correspond to those configured on the server as part of the "topp" workspace. 
+The above feature types correspond to those configured on the server as part of the ``topp`` workspace.
 
 The consequence of a virtual service is not only limited to the capabilities document of the service. When a client accesses a virtual service it is restricted to only those layers for all operations. For instance, consider the following WFS feature request::
 
   http://localhost:8080/geoserver/topp/wfs?request=GetFeature&typename=tiger:roads
 
-The above request results in an exception. Since the request feature type "tiger:roads" is not in the "topp" workspace the client will receive an error stating that the requested feature type does not exist. 
+The above request results in an exception. Since the request feature type ``tiger:roads`` is not in the ``topp`` workspace the client will receive an error stating that the requested feature type does not exist.
 
-Filtering by layer
-------------------
+Configuring workspace services
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Workspace services can be configured to override service configuration for an individual workspace.
+
+.. figure:: /data/webadmin/img/workspace_services.png
+   
+   Workspace services
+
+Each workspace service can be configureed with their own title, abstract and service settings. This can also be used to enable/disable specific services on a workspace by workspace basis to control what virtual services are available.
```

Comment:
```suggestion
Each workspace service can be configured with their own title, abstract and service settings. This can also be used to enable/disable specific services on a workspace by workspace basis to control what virtual services are available.
```